File: codigo/frontend_mobile/lib/models/reserva.dart
class Reserva {
  final String id;
  final String horarioChegadaEsperada;
  final String dataReserva;
  final int cliente;
  final int avaliacao;
  final String status;
  final List mesas;
  final List turnos;

  const Reserva({
    this.id,
    this.horarioChegadaEsperada,
    this.dataReserva,
    this.cliente,
    this.avaliacao,
    this.status,
    this.mesas,
    this.turnos,
  });
}

File: codigo/backend/ReserveAqui.Infra.Data/Entities/Reserva.cs
using System;
using System.Collections.Generic;

namespace ReserveAqui.Infra.Data.Entities
{
    public class Reserva
    {
        public int Id { get; set; }
        public TimeSpan HorarioChegadaEsperada { get; set; }
        public DateTime DataReserva { get; set; }
        public int IdCliente { get; set; }
        public int? Avaliacao { get; set; }
        public int IdStatus { get; set; }
        public List<int> IdMesas { get; set; }
        public List<int> IdTurnos { get; set; }
    }
}
